lib/google/ads/google_ads/google_ads_client.rb

```diff
@@ -96,6 +97,25 @@ def configure(&block)
           yield @config
         end
 
+        def configure_using_environment_variables
```

Comment:
Can we call this "load_environment_config" instead? I think "configure" implies that the user should be passing something, but "load" implies that there is some information already stored somewhere, which is more accurate.

Reply:
Another option would be something like "load_from_environment_variables"

Reply:
Makes sense. Changed it to load_from_environment_variables

Reply:
This is a method on the GoogleAdsClient, though, so "client.load_from_environment_variables" actually doesn't make as much sense as "client.load_config_from_environment_variables", but that's super long, which is why I originally proposed "client.load_environment_config" which is more reasonable to type. We have to specify *what* we're loading because the client itself isn't loading, one of its sub-pieces is.

Reply:
load_environment_config SGTM. Changed it accordingly. Thanks.